modules/client-configuration/pages/supported-features-amazon.adoc

```diff
@@ -0,0 +1,160 @@
+[[supported-features-amazon]]
+= Supported {amazonlinux} Features
+
+
+This table lists the availability of various features on {amazonlinux} clients.
+
+[IMPORTANT]
+====
+The operating system you run on a client is supported by the organization that supplies the operating system.
+{amazonlinux} is supported by Amazon.
+====
+
+The icons in this table indicate:
+
+* {check} the feature is available on both Salt and traditional clients
+* {cross} the feature is not available
+* {question} the feature is under consideration, and may or may not be made available at a later date
+* {trad} the feature is supported only on traditional clients
+* {salt} the feature is supported only on Salt clients
+
+
+[cols="1,1", options="header"]
+.Supported Features on {amazonlinux} Operating Systems
+|===
+
+| Feature
+| {amazonlinux}{nbsp}2
+
+| Client
+| {salt}
+
+| Operating system packages
+| {salt}
+
+| Registration
+| {salt}
+
+| Install packages
+| {salt}
+
+| Apply patches (requires CVE ID)
+| {question}
+
+| Remote commands
+| {salt}
+
+| System package states
+| {salt}
+
+| System custom states
+| {salt}
+
+| Group custom states
+| {salt}
+
+| Organization custom states
+| {salt}
+
+| System set manager (SSM)
+| {salt}
+
+| Service pack migration
+| N/A
+
+| Basic Virtual Guest Management {star}
+| {salt}
+
+| Advanced Virtual Guest Management {star}
+| {salt}
+
+| Virtual Guest Installation (Kickstart), as Host OS
+| {cross}
+
+| Virtual Guest Installation (image template), as Host OS
+| {question}
+
+| System deployment (PXE/Kickstart)
+| {question}
+
+| System redeployment (Kickstart)
+| {question}
+
+| Contact methods
+| {salt}: ZeroMQ, Salt-SSH
+
+| Works with {productname} Proxy
+| {salt}
+
+| Action chains
+| {salt}
+
+| Staging (pre-download of packages)
+| {salt}
+
+| Duplicate package reporting
+| {salt}
+
+| CVE auditing (requires CVE ID)
+| {question}
+
+| SCAP auditing
+| {question}
+
+| Package verification
+| {cross}
+
+| Package locking
+| {cross}
+
+| System locking
+| {cross}
+
+| Maintenance Windows
+| {check}
+
+| System snapshot
+| {cross}
+
+| Configuration file management
+| {salt}
+
+| Snapshots and profiles
+| {salt}: Profiles supported, Sync not supported
+
+| Power management
```

Comment:
I'd say "N/A" because by power management we mean IPMI or RedFish, which AFAIK are not a thing for virtual machines. @cbosdo can enlighten us here.

Reply:
There are no IPMI / Redfish implementation for libvirt at least.... I have no idea though if there is one for Amazon Linux VMs

Reply:
Keep in mind: https://aws.amazon.com/about-aws/whats-new/2019/02/introducing-five-new-amazon-ec2-bare-metal-instances/ I never tried this, but I am quite sure they use (or at least support using) Amazon Linux. No idea if Power Management works.

Reply:
Changed to question. We need to verify what happens with the EC2 bare metal instnaces.